bin/debug50

```diff
@@ -51,8 +51,8 @@ sources=$(echo "$sources" | grep -Eo "/.*\.c,?")
 
 # ensure executable is more recent than source and header files
 breakpoint=false
-IFS=$'\n'
-for f in ${sources//,/}
+IFS=$', '
+for f in $sources
```

Comment:
Why the need for the change, do you know?

Reply:
Added the `\n` back per https://github.com/cs50/harvard.cs50.debug/pull/25.